sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubOptions.cs

```diff
@@ -51,7 +45,7 @@ public EventHubConnectionOptions ConnectionOptions
         ///   amount of time allowed for receiving event batches and other interactions with the Event Hubs service.
         /// </summary>
         ///
-        public EventHubsRetryOptions RetryOptions
+        public EventHubsRetryOptions ClientRetryOptions
```

Comment:
Is this explicit enough? I though the suggestion was EventHubClientRetryOptions or something along those lines?

Reply:
I've tried it and it looked very verbose. I'll follow up with Paul.